app/controllers/users_controller.rb

```diff
@@ -20,9 +21,12 @@ def create
     end
   end
 
+  def show; end
+
   def destroy
+    return  redirect_to users_path, alert: 'Permission Denied - Cannot delete another User' unless @allow_action
     @user.destroy
-    redirect_to users_path
+    redirect_to users_path, notice: 'User deleted successfully'
```

Comment:
If the user is not deleted, this should send the same message?
Please add a conditional to handle the messages in order to notify when an error occurred.

Reply:
Devise is handling sessions!